content/en/blog/_posts/2023-07-24-unknown-version-interoperability-proxy-alpha.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: blog
+title: "Kubernetes 1.28: Upgrade Kubernetes more safely by enabling the Unknown Version Interoperability Proxy (Alpha)"
```

Comment:
I realize I'm opening a can of worms here, but do we always want to refer to this feature by the full "Unknown Version Interoperability Proxy" name? What if we refereed it to as the "Interoperability Proxy" in most documentation?

I suggest this for a few reasons:
- It's quite plausible we might introduce version information into the proxy in the future, invalidating the first part of the feature name
- I don't think users care or need to care about the fact that the interoperability is version unaware.

Reply:
That's legit.

I think if we're going to rename it, we should do that quite soon.
It's really an Unknown GVK Interoperability Proxy :grimacing: but let's not call it the UGVKIP.

How about calling this _Cluster Upgrade Interoperability_?

Reply:
(names welcome, but there is a deadline for these to be useful)

Reply:
Sorry for the delay on the action around this! Can I still update the name now?

Reply:
> How about calling this Cluster Upgrade Interoperability?

The feature also applies in cases where different API servers have different APIs enabled, not just when there is a cluster upgrade involved (though this would be the prime use case for this feature). Although not sure if such a scenario can exist in real world.

Reply:
I like `Mixed Version Proxy`. MVP for short :)

Reply:
I like getting rid of the Interoperability term. That term itself makes the whole feature name a bit mouthful.

Reply:
Updated text to use Mixed Version Proxy. Thanks @logicalhan for the suggestion!

Reply:
Thanks all!